README.md

```diff
@@ -27,30 +29,139 @@ npm install @dojo/widgets
 ```
 This package contains *all* of the widgets in this repo.
 
-All of the widgets in this repo are on the same release schedule, that is to say, that we release all widgets at the same time. Minor releases may include new widgets and/or features, whereas patch releases may contain fixes to more than 1 widget.
+All of the widgets are on the same release schedule, that is to say, that we release all widgets at the same time.
+Minor releases may include new widgets and/or features, whereas patch releases may contain fixes to more than 1 widget.
 
-In a future release, we will also publish each widget individually to `npm`.
+To use a widget in your application, you will need to import each widget individually:
+
+```ts
+import Button from '@dojo/widgets/button/Button';
+```
+
+Each widget module has a default export of the widget itself, as well as named exports for things such as properties specific to the widget:
+
+```ts
+import Button, { ButtonProperties } from '@dojo/widgets/button/Button';
+```
+
+Because each widget is a separate module, when you create a release build of your application, you will only include the widgets that you have explicitly imported.
+This allows our build tooling ([`dojo cli`](https://github.com/dojo/cli)), to make sure that the production build of your application only includes the widgets you use and is as small as possible. 
 
 ## Features
 
-Dojo 2 widgets are supported in all evergreen browsers (IE11+, Chrome, Firefox, Safari) as well as popular mobile browsers (Mobile Safari, Chrome on Android).
+- All widgets are supported in all evergreen browsers (Chrome, Edge, Firefox, IE11+ and Safari) as well as popular mobile browsers (Mobile Safari, Chrome on Android).
+
+- All widgets support accessibility (`a11y`) out of the box.
+
+- All widgets are fully themeable.
+In a future release we will ship with several official Dojo 2 themes that can be applied to the widgets.
+
+- All widgets support internationalization (`i18n`)
+
+## Widgets
+
+### Form widgets
+[Button](src/button/README.md)
+
+[Checkbox/Switch](src/checkbox/README.md)
 
-Dojo 2 widgets are fully accessible (`a11y`), out of the box.
+[ComboBox](src/combobox/README.md)
 
-Dojo 2 widgets are fully themeable. In a future release we will ship with several official Dojo 2 themes that can be applied to the widgets (all themes will live in a separate repo).
+[Label](src/label/README.md)
 
-Dojo 2 widgets are fully internationalisable (`i18n`)
+[Radio](src/radio/README.md)
+
+[Select](src/select/README.md)
+
+[Slider](src/slider/README.md)
+
+[TextArea](src/TextArea/README.md)
+
+[TextInput](src/TextInput/README.md)
+
+### Layout widgets
+[SlidePane](src/slidepane/README.md)
+
+[TabPane](src/tabpane/README.md)
+
+### Misc widgets
+[Dialog](src/dialog/README.md)
 
 ## Conventions
 
+### EventHandlers
```

Comment:
Do you think we need to go into this much detail about our naming conventions for event handling properties here? Part of me feels like this should be somewhere else. This sort of seems like it's meant for future Dojo 2 widget authors instead of downstream users. Maybe this could go under "Coding Conventions"?